code/__DEFINES/~yogs_defines/mobs.dm

```diff
@@ -0,0 +1,13 @@
+#define PRETERNIS_LEVEL_FULL 550
+#define PRETERNIS_LEVEL_WELL_FED 450
+#define PRETERNIS_LEVEL_FED 350
+#define PRETERNIS_LEVEL_HUNGRY 250
+#define PRETERNIS_LEVEL_STARVING 150
+#define PRETERNIS_LEVEL_NONE 0
+
+#define ELECTRICITY_TO_NUTRIMENT_FACTOR 0.12
+
+#define PRETERNIS_NV_OFF 2
```

Comment:
I'm assuming that this is how far you can see with NV on/off, should probably have a comment though